packages/lwc-language-server/package.json

```diff
@@ -23,6 +23,9 @@
     "package": "yarn pack"
   },
   "dependencies": {
+    "@komaci/static-analyzer": "238.1.4",
```

Comment:
Pulling in komaci.

This requires @komaci/types and @lwc-platform/lwc-metadata-next to be explicitly brought in, too, instead of being a transient dependency. Not sure if there is a better way to do this?

I also noticed 238.1.5 appears to be in the making, so we can update when it is available.

Reply:
Also, I believe that the komaci libraries are not publicly available, so is it even OK to merge this yet?